terraform/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "tempo"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = "latest/stable"
```

Comment:
Same comment as in https://github.com/canonical/observability/pull/209#discussion_r1782428694 : maybe it's worth a follow-up PR once we release to stable?